level.go

```diff
@@ -0,0 +1,43 @@
+package corelog
+
+import "log/slog"
+
+const (
+	levelDebug = slog.LevelDebug
+	levelInfo  = slog.LevelInfo
+	levelError = slog.LevelError
+	levelFatal = slog.Level(12)
+)
+
+// levelLabels is a mapping of log levels to custom labels.
+var levelLabels = map[slog.Level]string{
+	levelDebug: "DEBUG",
```

Comment:
question: Do we actually wish to define `DEBUG` and `FATAL` at Source?

Reply:
I am in favor of removing both, but happy to hear more opinions.

Reply:
Removed both `DEBUG` and `FATAL` after discussing at the standup